components/bin/PartTag.js

```diff
@@ -0,0 +1,151 @@
+import React from 'react'
+import styles from './PartTag.module.css'
+import { useState } from 'react';
+
+const PartTag = ({ partID, search = false, addFilter, removeFilter}) => {
+  const [isOutlined, setIsOutlined] = useState(false);
+
+  const handleClick = () => {
+    if (search){
+      setIsOutlined(prevState => !prevState);
+      if (isOutlined){
+        removeFilter(partID);
+      } else {
+        addFilter(partID);
+      }
+    }
+    
+  };
+
```

Comment:
This switch statement is hella ugly, and if the Bin ever moves to a different owner and the list of parts changes it will confound that maintainer to have this second source of truth here. Clay and I talked about this and it seems unlikely to become a real issue. @maxwofford I defer to you on whether or not this is something actually worth changing.